sycl/include/CL/sycl/exception.hpp

```diff
@@ -23,18 +23,34 @@ namespace sycl {
 // Forward declaration
 class context;
 
+#define Exception121DeprecationMessage "Use SYCL 2020 exceptions."
```

Comment:
1. Please, `undef` the macro at the end of the file.
2. Is it possible to provide a user with a more specific suggestion? Like "*This* class is deprecated, use `sycl::exception` instead` ?

Reply:
C++ core guidelines say "ES.31: Don’t use macros for constants or “functions”" - 
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es31-dont-use-macros-for-constants-or-functions

Reply:
I removed the macro.  A typical error message looks like this now:
`'kernel_error' is deprecated: use sycl::exception with sycl::errc::kernel or errc::kernel_argument instead.`